modules/insights-operator-configuring-sca.adoc

```diff
@@ -4,9 +4,9 @@
 
 
 [id="insights-operator-configuring-sca_{context}"]
-= Configuring Simple Content Access import interval
+= Configuring simple content access import interval
 
-You can configure how often the Insights Operator imports the RHEL Simple Content Access (SCA) certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you may want to shorten this interval if you update your SCA configuration in Red Hat Subscription Management.
+You can configure how often the Insights Operator imports the simple content access certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you may want to shorten this interval if you update your simple content access configuration in Red Hat Subscription Management.
```

Comment:
I know this is existing text, but can you s/may/might? Or maybe better, _but you can shorten..._.  
Or even two sentences? 
_The certificate import normally occurs every 8 hours. You can shorten..._

Reply:
Updated -made more sense to keep it one sentence- , but you can shorten...